src/layouts/SessionsSection.jsx

```diff
@@ -82,6 +82,9 @@ const SessionsSection = ({
           className="h-12 md:h-14 lg:h-16"
         />
         <button
+          aria-label={
+            isExpanded ? 'Collapse 2025 Sessions' : 'Expand 2025 Sessions'
```

Comment:
Thanks for adding this in - I missed this. Can you please adjust this to dynamically account for the year with the `year` variable in a template string as this will change